.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -157,6 +161,7 @@ jobs:
             --set=ipv4.enabled=true \
             --set=ipv6.enabled=true \
             --set=clustermesh.useAPIServer=${{ !matrix.external-kvstore }} \
+            --set=clustermesh.maxConnectedClusters=${{ matrix.max-connected-clusters }} \
```

Comment:
Commenting here, but the comment actually refers to line 276.

I'd suggest replacing the hard-coded 255 ClusterID value down there with `${{ matrix.max-connected-clusters }}`, so that the tests are performed using the two extreme ClusterIDs.

Reply:
I like where your head's at :+1: